apis/core/test/domain/column-mapping/create.test.ts

```diff
@@ -104,4 +116,23 @@ describe('domain/column-mapping/create', () => {
       }],
     })
   })
+
+  it('creates Dimension Metadata for column', async () => {
+    // given
+    const resource = clownface({ dataset: $rdf.dataset() })
+      .node($rdf.namedNode(''))
+      .addOut(cc.sourceColumn, $rdf.namedNode('my-column'))
+      .addOut(cc.targetProperty, $rdf.namedNode('test'))
+
+    // when
+    await createColumnMapping({ resource, store, tableId: table.term, dimensionMetadataQueries })
+
+    // then
+    expect(dimensionMetadata).to.matchShape({
+      property: [{
+        path: schema.hasPart,
+        minCount: 1,
+      }],
```

Comment:
You can also use `sh:node` to check deeper into the graph

```suggestion
      property: [{
        path: schema.hasPart,
        minCount: 1,
        node: {
          property: {
            path: schema.about,
            minCount: 1,
            maxCount: 1,
          }
        }
      }],
```